ios/sdk/src/JitsiMeetView.m

```diff
@@ -136,6 +141,20 @@ - (void)loadURL:(NSURL *)url {
 - (void)initialize {
     static dispatch_once_t onceToken;
 
+    /*
+     * TODO: Only allow a single instance for now. All React Native modules are
+     * kinda singletons so global state would be broken since we have a single
+     * bridge. Once we have that sorted out multiple instances of JitsiMeetView
+     * will be allowed.
+     */
+    if (instance != nil) {
+        @throw [NSException
+            exceptionWithName:@"RuntimeError"
+                       reason:@"Only a single instance is currently allowed"
+                     userInfo:nil];
+    }
+    instance = self;
```

Comment:
On both Android and iOS you only ever assign non-`null` to the static `mInstance` and `instance`, respectively. Consequently, that precludes the scenario of 2 `JitsiMeetView`s which do NOT exist simultaneously.

While I do understand that it's very unlikely that the scenario with 2 existing `JitsiMeetView`s which simultaneously drive 2 conferences will work because of non-shareable resources which we cannot make shareable, I think the scenario of multiple non-simultaneous `JitsiMeetView`s is likely.